[2025-03-17T18:55:17.930+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T18:55:18.063+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.user_sessions_project_c_db manual__2025-03-17T18:55:12.630245+00:00 [queued]>
[2025-03-17T18:55:18.220+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.user_sessions_project_c_db manual__2025-03-17T18:55:12.630245+00:00 [queued]>
[2025-03-17T18:55:18.224+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T18:55:18.371+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.user_sessions_project_c_db> on 2025-03-17 18:55:12.630245+00:00
[2025-03-17T18:55:18.472+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2264) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T18:55:18.512+0000] {standard_task_runner.py:72} INFO - Started process 2278 to run task
[2025-03-17T18:55:18.499+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.user_sessions_project_c_db', 'manual__2025-03-17T18:55:12.630245+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpvat2fgr4']
[2025-03-17T18:55:18.530+0000] {standard_task_runner.py:105} INFO - Job 127: Subtask load_tables.user_sessions_project_c_db
[2025-03-17T18:55:18.893+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.user_sessions_project_c_db manual__2025-03-17T18:55:12.630245+00:00 [running]> on host c032966e105e
[2025-03-17T18:55:19.684+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.user_sessions_project_c_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T18:55:12.630245+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T18:55:12.630245+00:00'
[2025-03-17T18:55:19.710+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T18:55:19.862+0000] {sql.py:278} INFO - Executing: select load_tables('user_sessions_project_c_db', 'user_sessions');
[2025-03-17T18:55:19.990+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T18:55:20.243+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T18:55:20.324+0000] {sql.py:553} INFO - Running statement: select load_tables('user_sessions_project_c_db', 'user_sessions');, parameters: None
[2025-03-17T18:55:20.600+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T18:55:21.103+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T18:55:21.133+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.user_sessions_project_c_db, run_id=manual__2025-03-17T18:55:12.630245+00:00, execution_date=20250317T185512, start_date=20250317T185518, end_date=20250317T185521
[2025-03-17T18:55:21.374+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T18:55:21.482+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T18:55:21.512+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
